### DISCUSSION TITLE
Unable to launch Winmerge Portable

### DISCUSSION THREAD

**Discussion by RG255** [WinMerge/winmerge#1625] [Q&A] [answered]
![Screenshot 2022-12-21 163055](https://user-images.githubusercontent.com/60707705/208959807-6cec8e3e-27ba-4c9c-be41-f8f1c6d89bb5.png)

WinmergePortable is giving be the above error on launch. Any ideas as to why this may be please.

Thanks

**Comment by sdottaka** [comment, marked as answer]
The problem seems to be with WinMergePortable.exe rather than with WinMerge itself.

Have you looked at https://portableapps.com/node/62594?

**Reply by RG255** [reply]
Thanks - that does answer the question.
